.github/workflows/test.yml

```diff
@@ -12,19 +12,22 @@ jobs:
     runs-on: ${{ matrix.os }}
 
     steps:
-    - name: Check out Git repository
```

Comment:
Unrelated to PR but should we make `.yml` and `.md` files be formatted by prettier?